plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1348,6 +1349,14 @@ blockly.block.entity_iterator=Entity iterator
 blockly.block.entity_none=No entity
 blockly.block.damagesource_from_deps=Provided damage source
 blockly.block.damagesource_from_type=Damage source of type: %1
+blockly.block.damagesource_from_type_cause=Damage source of type: %1 caused by: %2
+blockly.block.damagesource_from_type_cause_directcause=Damage source of type: %1 caused by: %2 direct cause: %3
+blockly.block.damagesource_get_direct_entity=%2 Get entity that directly caused %1
+blockly.block.damagesource_get_entity=%2 Get entity that caused %1
+blockly.block.damagesource_get_exhaustion=Get exhaustion value of %1
+blockly.block.damagesource_is_cause_creative=Is %1 caused by a player in creative mode
+blockly.block.damagesource_is_indirect=Is %1 indirect
```

Comment:
What is called an indirect damage?

Reply:
Oh ok, I understand now 👍
(Maybe explaining this in a tooltip could be helpful for some users)